web/src/app/users/UserContactMethodListDest.tsx

```diff
@@ -0,0 +1,265 @@
+
+  const getIcon = (cm: UserContactMethod): JSX.Element | null => {
+    if (!cm.disabled) return null
+    if (props.readOnly) {
+      return <Warning message='Contact method disabled' />
+    }
+
+    return (
+      <IconButton
+        data-cy='cm-disabled'
+        aria-label='Reactivate contact method'
+        onClick={() => setShowVerifyDialogByID(cm.id)}
+        disabled={props.readOnly}
+        size='large'
+      >
+        <Warning message='Contact method disabled' />
+      </IconButton>
+    )
+  }
+
+  function getActionMenuItems(cm: UserContactMethod): ListItemAction[] {
+    const actions = [
+      {
+        label: 'Edit',
+        onClick: () => setShowEditDialogByID(cm.id),
+        disabled: false,
+        tooltip: '',
+      },
+      {
+        label: 'Delete',
+        onClick: () => setShowDeleteDialogByID(cm.id),
+        disabled: false,
+        tooltip: '',
+      },
+    ]
+
+    // disable send test and reactivate if not current user
+    if (!cm.disabled) {
+      actions.push({
+        label: 'Send Test',
+        onClick: () => setShowSendTestByID(cm.id),
+        disabled: !isCurrentUser,
+        tooltip: !isCurrentUser
+          ? 'Send Test only available for your own contact methods'
+          : '',
+      })
+    } else {
+      actions.push({
+        label: 'Reactivate',
+        onClick: () => setShowVerifyDialogByID(cm.id),
+        disabled: !isCurrentUser,
+        tooltip: !isCurrentUser
+          ? 'Reactivate only available for your own contact methods'
+          : '',
+      })
+    }
+    return actions
+  }
+
+  function getSecondaryAction(cm: UserContactMethod): JSX.Element {
+    return (
+      <Grid container spacing={2} alignItems='center' wrap='nowrap'>
+        {cm.disabled && !props.readOnly && !mobile && isCurrentUser && (
+          <Grid item>
+            <Button
+              aria-label='Reactivate contact method'
+              onClick={() => setShowVerifyDialogByID(cm.id)}
+              variant='contained'
+            >
+              Reactivate
+            </Button>
+          </Grid>
+        )}
+        {!props.readOnly && (
+          <Grid item>
+            <OtherActions actions={getActionMenuItems(cm)} />
+          </Grid>
+        )}
+      </Grid>
+    )
+  }
+
+  function getSubText(cm: UserContactMethod): JSX.Element {
+    return (
+      <React.Fragment>
+        {cm.dest.values.map((v) => {
+          let cmText = v.label
+          if (cm.pending) {
+            cmText = `${cmText} - this contact method will be automatically deleted if not verified`
+          }
+          if (v.fieldID === 'webhook-url') {
+            return (
+              <Typography key={v.toString()}>
+                {`${cmText} (`}
+                <AppLink to='/docs#webhooks'>docs</AppLink>)
+              </Typography>
+            )
+          }
+
+          return <Typography key={v.toString()}>{cmText}</Typography>
+        })}
+      </React.Fragment>
+    )
```

Comment:
We cannot hard-code anything like `webhook-url` we need to rely on the `displayInfo` prop entirely which already includes the link and link text when necessary.